### ISSUE TITLE
Implement session affinity for ingress

### ISSUE THREAD

**Issue by davidfowl** [dotnet/tye#257]
We need one more type of load balancing besides round robin, session affinity. It's important for things like SignalR and Blazor Server (which uses SignalR).

**Comment by Nordes** [comment]
Yes, this is an enabler for sticky session. It would be a nice a to have.
